test/glific_web/providers/maytapi/controllers/message_controller_test.exs

```diff
@@ -366,26 +346,19 @@ defmodule GlificWeb.Providers.Maytapi.Controllers.MessageControllerTest do
       bsp_message_id = get_in(@media_message_webhook, ["message", "id"])
 
       {:ok, message} =
-        Repo.fetch_by(Message, %{
-          bsp_message_id: bsp_message_id,
+        Repo.fetch_by(WAMessage, %{
+          bsp_id: bsp_message_id,
           organization_id: conn.assigns[:organization_id]
         })
 
-      message = Repo.preload(message, [:receiver, :sender, :media, :contact])
+      message = Repo.preload(message, [:media, :contact])
 
       # Provider message id should be updated
       assert message.bsp_status == :delivered
       assert message.flow == :inbound
 
       # ensure the message has been received by the mock
-      assert_receive :received_message_to_process
-
-      assert message.sender.last_message_at != nil
-      assert true == Glific.in_past_time(message.sender.last_message_at, :seconds, 10)
-
-      # Sender should be stored into the db
-      assert message.sender.phone ==
-               get_in(@media_message_webhook, ["user", "phone"])
+      # assert_receive :received_message_to_process
```

Comment:
do we need this?

Reply:
This is a test where we create a contact when we receive_message via gupshup and then again receive message via maytapi. So above lines are gupshup related